packages/perseus/src/widgets/matrix/score-matrix.test.ts

```diff
@@ -1,13 +1,66 @@
 import {mockStrings} from "../../strings";
 
 import scoreMatrix from "./score-matrix";
+import * as MatrixValidator from "./validate-matrix";
 
 import type {
     PerseusMatrixRubric,
     PerseusMatrixUserInput,
 } from "../../validation.types";
 
 describe("scoreMatrix", () => {
+    it("should be correctly answerable if validation passes", function () {
+        // Arrange
+        const mockValidator = jest
+            .spyOn(MatrixValidator, "default")
+            .mockReturnValue(null);
```

Comment:
Question: I'm curious: why do we need to spy on the validator and give a mock return value here? The validator is called as part of the`scoreMatrix` function, right? Ooo, or is this to bypass the actual validation function so the test doesn't take that function into account? Should this be done with all the other scoring tests?

Reply:
[Jeremy left a comment about this on one of my PRs](https://github.com/Khan/perseus/pull/1876/files/a9351e96858891d2fa6bb178527ad5758429b83c#r1847397738)